front-end/src/app/profile/profile.component.html

```diff
@@ -0,0 +1,223 @@
+<h1>Account Info</h1>
+<p-divider></p-divider>
+<div>
+  <div class="grid">
+    <div class="col-12">
+      <h3>Committee Info</h3>
+    </div>
+  </div>
+  <div class="grid">
+    <div class="col-3">
+      <div class="field">
+        <h4>COMMITTEE ID</h4>
+        <p>
+          {{ (committeeAccount$ | async)?.committee_id }}
+        </p>
+      </div>
+    </div>
+    <div class="col-3">
+      <div class="field">
+        <h4>COMMITTEE NAME</h4>
+        <p>
+          {{ (committeeAccount$ | async)?.name }}
+        </p>
+      </div>
+    </div>
+    <div class="col-3">
+      <div class="field">
+        <h4>TYPE OF COMMITTEE</h4>
+        <p>
+          {{ (committeeAccount$ | async)?.committee_type_full }}
+        </p>
+      </div>
+    </div>
+  </div>
+</div>
+
+<p-divider></p-divider>
+
+<div>
+  <div class="grid">
+    <div class="col-12">
+      <h3>Committee Address</h3>
+    </div>
+  </div>
+  <div class="grid">
+    <div class="col-3">
+      <div class="field">
+        <h4>ADDRESS</h4>
+        <p>
+          {{ (committeeAccount$ | async)?.street_1 }}
+        </p>
+      </div>
+    </div>
+    <div class="col-9">
+      <div class="field">
+        <h4>APPARTMENT, SUITE, ETC.</h4>
+        <p>
+          {{ (committeeAccount$ | async)?.street_2 }}
+        </p>
+      </div>
+    </div>
+    <div class="col-3">
+      <div class="field">
+        <h4>CITY</h4>
+        <p>
+          {{ (committeeAccount$ | async)?.city }}
+        </p>
+      </div>
+    </div>
+    <div class="col-3">
+      <div class="field">
+        <h4>STATE/PROVINCE</h4>
+        <p>
+          {{ (committeeAccount$ | async)?.state }}
+        </p>
+      </div>
+    </div>
+    <div class="col-6">
+      <div class="field">
+        <h4>ZIP/POSTAL CODE</h4>
+        <p>
+          {{ (committeeAccount$ | async)?.zip }}
+        </p>
+      </div>
+    </div>
+    <div class="col-3">
+      <div class="field">
+        <h4>EMAIL</h4>
+        <p>
+          {{ (committeeAccount$ | async)?.email }}
+        </p>
+      </div>
+    </div>
+    <div class="col-9">
+      <div class="field">
+        <h4>WEBSITE</h4>
+        <p>
+          {{ (committeeAccount$ | async)?.website }}
+        </p>
+      </div>
```

Comment:
![app210-email-website-overlap](https://user-images.githubusercontent.com/2971785/162080368-35ca3fc4-09e9-4a16-9bf2-fd0f5d0e102c.png)
The email and website content is overlapping